src/main/java/ua/tc/marketplace/controller/UserController.java

```diff
@@ -0,0 +1,59 @@
+package ua.tc.marketplace.controller;
+
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import ua.tc.marketplace.model.dto.user.CreateUserDto;
+import ua.tc.marketplace.model.dto.user.UpdateUserDto;
+import ua.tc.marketplace.model.dto.user.UserDto;
+import ua.tc.marketplace.service.UserService;
+import ua.tc.marketplace.util.openapi.UserOpenApi;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api/v1/user")
+public class UserController implements UserOpenApi {
+
+  private final UserService userService;
+
+  @Override
+  @GetMapping("/all")
+  public ResponseEntity<Page<UserDto>> getAllUsers(@PageableDefault Pageable pageable) {
+    return ResponseEntity.status(HttpStatus.OK).body(userService.findAll(pageable));
+  }
+
+  @GetMapping("/{id}")
+  public ResponseEntity<UserDto> getUserById(@PathVariable Long id) {
+    return ResponseEntity.status(HttpStatus.OK).body(userService.findUserById(id));
+  }
+
+  @PostMapping()
+  public ResponseEntity<UserDto> createUser(@Valid @RequestBody CreateUserDto userDto) {
+    return ResponseEntity.status(HttpStatus.OK).body(userService.createUser(userDto));
+  }
+
+  @PutMapping()
```

Comment:
Here will be a discussion. You use the approach when id of updated user is in request body. I use request body without id and id as path variable in my controllers (adController, photoController). I think we should act in one style. Lets discuss in team chat what is the best

Reply:
agree, we need to discuss that and workout common aproach